exchange/howto/run-seednode.adoc

```diff
@@ -4,108 +4,306 @@
-But more critical is that if a node has bad connectivity it might miss messages like mailbox messages. That would be very bad because if a trader connects to that node he will not see the message the peer or arbitrator has sent him when he was offline. Usually we never had issues with that but it could become a cause for disputes or bug reports if a node would be badly connected.
+. Deliver the peer addresses of other dynamic network peers
 
+. Deliver the initial P2P network data like open offers, mailbox messages and arbitrators
+
+Being the gateways to the Bisq P2P network, seed nodes are among the first nodes a Bisq client connects to on each startup. Therefore, seed nodes have distinct properties:
+
+- Seed nodes face a heavy load and are therefore operated on servers offering a high level of reliability and better connectivity than a normal Bisq client.
+- A bad seed node can have a substantial impact on the Bisq network. A Bisq user may encounter anything from a longer startup time of her Bisq client (which is unconvenient but not critical) to missing messages sent to her by a trading partner or arbitrator while her Bisq client has been offline (which can lead to disputes, failed trades, blocking of the Bisq client and bad reputation of the Bisq client and therefore the user). 
+
+All in all, seed nodes are important for the Bisq network to function in a user-friendly way. Without the seed nodes, a fresh Bisq client would not find its way into the network, at least not without manual intervention. Bad seed nodes can make it harder for a fresh client to join the network. In the end, seed nodes allow new users easy access to the services of Bisq and should therefore be kept up and running.
 
 == Seed node operators
 
-The seed node addresses (onion address) are hard coded in the application but can be overruled if the user adds a seed node address as program argument. Any user can run therefor their own seed node and connect to it.
+Being a seed node operator brings along duties and benefits making it worthwhile.
 
-Some contributors of Bisq are running the default seed nodes which are hard coded in the application. It requires a to set up a bond in BSQ to get the privilege to run a default seed node and it requires that a certain quality of service is met. The node needs to support at least 30 connections.
+=== Duties of the seed node operator
 
-Contributors running a seed node will file a compensation request each month and get paid a fixed amount of BSQ for it if node availability was as expected.
+The operator of a seed node needs to:
 
-A default seed node can be blocked by a network message in case the seed node does not fulfill the required quality.
+- find yourself a deputy in case you are on not available (travel, holiday, others, ...). Usually, on of the developers is there to help. Just inform them on your setup and all is good.
```

Comment:
```suggestion
- find yourself a deputy in case you are not available (travel, holiday, others, ...). Usually, one of the developers is there to help. Just inform them on your setup and all is good.
```